src/views/Restoration.vue

```diff
@@ -207,6 +225,26 @@ export default class Restoration extends Vue {
 
       // fetch entity snapshot
       const entitySnapshot = await this.fetchEntitySnapshot()
+      const stateFiling = entitySnapshot.businessInfo.stateFiling
+      const filing = stateFiling && await LegalServices.fetchFiling(stateFiling)
+
+      if (!filing) {
+        throw new Error(`Invalid fetched stateFiling = ${this.getBusinessId}`)
+      }
+
+      const parties = filing.restoration?.parties || []
+
+      // find first applicant from fetched parties
+      const applicants = parties.find(
+        orgPerson => orgPerson.roles.some(role => role.roleType === RoleTypes.APPLICANT)
+      )
+
+      if (applicants === undefined) {
+        throw new Error(`Applicant not found for ${this.getBusinessId}`)
+      }
+
```

Comment:
this will handle if parties and applicants are undefined. I tested it and error thrown will be like below: 

![image](https://user-images.githubusercontent.com/116035339/224845418-f425e967-d3ce-402c-9a1b-8a76bdd94c96.png)